### PR TITLE
docs: Add instruction how to trigger and clone tests

### PR DESCRIPTION
**Pull request by baierjan**
Reference: https://progress.opensuse.org/issues/155491

### CODE REVIEW

**Review by kalikiana — APPROVED** [review]
[Looks nice!](https://github.com/os-autoinst/openQA/blob/f7bfa444e211e741811dc0aaabe67e9e0bd20671/docs/Installing.asciidoc#how-to-run-a-test-with-single-instance-container-in-5-minutes)

**Review by perlpunk — APPROVED** [review]

File: docs/Installing.asciidoc [hunk 1]
```diff
@@ -51,6 +51,35 @@ or https://localhost:1443.
...
+[source,sh]
+----
+openqa-cli schedule --monitor \
+    --param-file SCENARIO_DEFINITIONS_YAML=scenario-definitions.yaml \
+    DISTRI=example VERSION=0 FLAVOR=DVD ARCH=x86_64 \
+    TEST=simple_boot _GROUP_ID=0 BUILD=test \
+    CASEDIR=https://github.com/os-autoinst/os-autoinst-distri-example.git \
+    NEEDLES_DIR=%%CASEDIR%%/needles
+
+openqa-clone-job openqa-clone-job https://openqa.opensuse.org/tests/1896520
```
Comment (Martchus) [inline docs/Installing.asciidoc:77]:
```suggestion
openqa-clone-job https://openqa.opensuse.org/tests/1896520
```

File: docs/Installing.asciidoc [hunk 2]
```diff
@@ -51,6 +51,35 @@ or https://localhost:1443.
 
 image::images/openqa-in-5-minutes.gif[Running openQA job within 5 minutes]
 
+==== Triggering and cloning existing jobs within single-instance container
+
+For triggering new test or cloning existing one you can use `openqa-cli` which
```
Comment (Martchus) [inline docs/Installing.asciidoc:56]:
```suggestion
For triggering new tests or cloning existing ones you can use `openqa-cli` which
```

(or insert articles)

File: docs/Installing.asciidoc [hunk 3]
```diff
@@ -51,6 +51,35 @@ or https://localhost:1443.
...
+is conveniently available inside the container. The quickest and easiest way
+would be to enter an interactive session inside the already running
+single-instance container. You can spawn a new shell via:
+
+[source,sh]
+----
+podman exec -ti openqa /bin/bash
+----
+
+From there, you can trigger a new job or clone existing one, e.g.:
```
Comment (Martchus) [inline docs/Installing.asciidoc:66]:
```suggestion
From there, you can trigger a new job or clone an existing one, e.g.:
```

File: docs/Installing.asciidoc (hunk 1 above)
Comment (baierjan) [reply docs/Installing.asciidoc:77]: Ahh, I missed this copy&paste typo.. Will fix in a moment.

Comment (baierjan) [reply docs/Installing.asciidoc:77]: #5686

**Pull request merged**